.gitignore

```diff
@@ -24,8 +24,9 @@ ehthumbs.db
 Thumbs.db
 
 # Frontend #
-chatbot-ui/.next/
 node_modules
 tsconfig.tsbuildinfo
+chatbot-ui/.env.local
```

Comment:
minor comment: could probably just add `chatbot-ui/!.env.example` and then `chatbot-ui/.env.*` in order to ignore example files and to prevent people accidentally committing env files 😛

Reply:
Good suggestion, thanks! I've made this change now. Good to get other env files excluded. 🙂